index.html

```diff
@@ -0,0 +1,227 @@
+          <div class="muti-post flex-item">
+            <h3>Multi-Post Stories <br> Gain+Glory</h3>
+          </div>
+          <div class="language-skills">
+            <ul>
+              <li><a href="#">Ruby on rails</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">html</a></li>
+            </ul>
+          </div>
+          <div class="btn"><a href="#">See Project</a></div>
+        </div>
+      </div>
+
+      <!-- Sixth part -->
+      <div class="article">
+        <div class="first-part"></div>
+        <div class="second-part">
+          <div class="muti-post flex-item">
+            <h3>Multi-Post Stories <br> Gain+Glory</h3>
+          </div>
+          <div class="language-skills">
+            <ul>
+              <li><a href="#">Ruby on rails</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">html</a></li>
+            </ul>
+          </div>
+          <div class="btn"><a href="#">See Project</a></div>
+        </div>
+      </div>
+    </div>
+
+    <div class="about-me section">
+      <div class="article to-know">
+        <h3>About me</h3>
+        <div class="text">
+          I can help you build a product , feature or website Look through some of my work and experience! If you
+          like
+          what you see and have a project you need coded, don’t hestiate to contact me.
+        </div>
+        <div class="resume"><a href="#">Get My Resume</a></div>
+      </div>
+
+      <div class="article one details">
+        <div class="my-image">
+          <img src="images/languages.svg" alt="">
+        </div>
+        <h3>Languages</h3>
+        <ul>
+          <li class="js"><a href="#">JavaScript</a></li>
+          <li><a href="#">Ruby</a></li>
+          <li><a href="#">Html</a></li>
+          <li><a href="#">css</a></li>
+        </ul>
+      </div>
+
+      <div class="article two details">
+        <div class="my-image">
+          <img src="images/frameworks.svg" alt="">
+        </div>
+        <h3>Frameworks</h3>
+        <ul class="ul-1">
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby on rails</a></li>
+          <li><a href="#">RSpec</a></li>
+        </ul>
+        <ul class="ul-2">
+          <li><a href="#">Capybara</a></li>
+          <li><a href="#">Celenium</a></li>
+        </ul>
+      </div>
+
+      <div class="article three details">
+        <div class="my-image">
+          <img src="images/skills.png" alt="">
+        </div>
+        <h3>Skills</h3>
+        <ul class="ul-1">
+          <li><a href="#">Codekit</a></li>
+          <li><a href="#">Github</a></li>
+          <li><a href="#">Codepen</a></li>
+          <li><a href="#">Gitlab</a></li>
+        </ul>
+        <ul class="ul-2">
+          <li><a href="#">Terminal</a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] Good job on the card but please do the changes the previous reviewer asked you to do. The design of the page is still the same. Here you have a screenshot of the card:

![image](https://user-images.githubusercontent.com/74506933/154817699-fb5e7bd1-18cc-4bdd-97c1-a8bf4cbe2547.png)

Reply:
@dd-obua implementing these changes will make your project look like the original design, kindly make the requested changes 🤝